app/controllers/concerns/api_responders.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ApiResponders
+  extend ActiveSupport::Concern
+
+  private
+
+    def respond_with_error(message, status = :unprocessable_entity, context = {})
```

Comment:
Can we use named parameters? This way it will be more explicit when being called. Also default error should be `internal_server_error`

Reply:
Eg: `respond_with_success status: :created, message: t("successfully_created", entity: "Article")`